src/server.rs

```diff
@@ -398,7 +392,7 @@ impl<S, M> Handler for Server<S, M> where S: Store, M: StateMachine {
 
 impl <S, M> fmt::Debug for Server<S, M> where S: Store, M: StateMachine {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        write!(fmt, "Server({})", self.id)
+        fmt::Debug::fmt(&self.id, fmt)
```

Comment:
The ServerId is constant, but I think it makes for better log messages if this prints `Server` instead of `ServerId`.